public/js/login.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+function setTime() {
+  var now = new Date();
+  var time = now.toLocaleTimeString();
+  $('#time').html('Welcome, it is now ' + time);
+}
+
+function validateLoginForm() {
+  var username = $('#username').val();
+  var password = $('#password').val();
+
+  if (username === '') {
+    $('#username').css('border', '1px solid red');
+    $('#username_error').text('username is required');
+    $('#username').focus();
+    return false;
+  } else {
+    $('#username_error').text('');
+  }
+
+  if (password === '') {
+    $('#password').css('border', '1px solid red');
+    $('#password_error').text('password is required');
+    $('#password').focus();
+    return false;
+  } else {
+    $('#password_error').text('');
+  }
+}
+
+function loginToAccount() {
+  var username = $('#username').val();
+  var password = $('#password').val();
+
+  if (username && password) {
+    $.ajax({
+      url: 'js/credentials.json',
+      dataType: 'json',
+      type: 'GET',
+      cache: true,
+      success: function(isLoggedIn) {
```

Comment:
`isLoggedIn` is better, but usually a prefix of `is` means the variable represents a boolean value, which this does not. Please give the name another shot :)